cypress-tests/README.md

```diff
@@ -34,65 +81,68 @@ To run test cases, follow these steps:
+   To learn about the supported commands, execute:
 
    ```shell
-   export CYPRESS_CONNECTOR="connector_id"
-   export CYPRESS_BASEURL="base_url"
-   export DEBUG=cypress:cli
-   export CYPRESS_ADMINAPIKEY="admin_api_key"
-   export CYPRESS_CONNECTOR_AUTH_FILE_PATH="path/to/creds.json"
+   npm run
    ```
 
-5. Run Cypress test cases
-
-   To run the tests in interactive mode run the following command
+3. Set up the cards database:
 
    ```shell
-   npm run cypress
+   psql --host=localhost --port=5432 --username=db_user --dbname=hyperswitch_db --command "\copy cards_info FROM '.github/data/cards_info.csv' DELIMITER ',' CSV HEADER;"
    ```
 
-   To run all the tests in headless mode run the following command
+4. Set environment variables for cypress
 
    ```shell
-   npm run cypress:ci
+   export CYPRESS_CONNECTOR="connector_id"
+   export CYPRESS_BASEURL="base_url"
+   export DEBUG=cypress:cli
+   export CYPRESS_ADMINAPIKEY="admin_api_key"
+   export CYPRESS_CONNECTOR_AUTH_FILE_PATH="path/to/creds.json"
    ```
 
-   To run payment tests in headless mode run the following command
+> [!TIP]
+> It is recommended to install [direnv](https://github.com/direnv/direnv) and use a `.envrc` file to store these environment variables with `cypress-tests` directory. This will make it easier to manage environment variables while working with Cypress tests.
 
-   ```shell
-   npm run cypress:payments
-   ```
+> [!NOTE]
+> To learn about how `creds` file should be structured, refer to the [example.creds.json](#example-credsjson) section below.
 
-   To run payout tests in headless mode run the following command
+## Running Tests
 
-   ```shell
-   npm run cypress:payouts
-   ```
+Execution of Cypress tests can be done in two modes: Development mode (Interactive) and CI mode (Headless). The tests can be executed against a single connector or multiple connectors in parallel. Time taken to execute the tests will vary based on the number of connectors and the number of tests. For a single connector, the tests will take approximately 07-12 minutes to execute.
 
-   To run routing tests in headless mode run the following command
+### Development Mode (Interactive)
 
-   ```shell
-   npm run cypress:routing
-   ```
+```bash
+npm run cypress
+```
 
-In order to run cypress tests against multiple connectors at a time or in parallel:
+### CI Mode (Headless)
 
-1. Set up `.env` file that exports necessary info:
+```bash
+# All tests
+npm run cypress:ci
 
-   ```env
-   export DEBUG=cypress:cli
+# Specific test suites
+npm run cypress:payments            # Payment tests
+npm run cypress:payment-method-list # Payment method list tests
+npm run cypress:payouts             # Payout tests
+npm run cypress:routing             # Routing tests
+```
 
-   export CYPRESS_ADMINAPIKEY='admin_api_key'
-   export CYPRESS_BASEURL='base_url'
-   export CYPRESS_CONNECTOR_AUTH_FILE_PATH="path/to/creds.json"
+### Execute tests against multiple connectors or in parallel
```

Comment:
2nd item in this section:

> `scripts/execute_cypress.sh`

If we've already switched to the `cypress-tests` directory, then you can't access `scripts` from the directory right? We'd have to access it possibly as `../scripts/<...>`?

Reply:
ah, yes. i took this from the perspective of being in the root directory. updated.